src/core/ObservableQuery.ts

```diff
@@ -730,12 +731,18 @@ Did you mean to call refetch(variables) instead of refetch({ variables })?`);
 
     const info = pollingInfo || (this.pollingInfo = {} as any);
     info.interval = pollInterval;
+  
+    // The fetchPolicy to be used when polling. Defaults to network-only.
+    let pollFetchPolicy:PollFetchPolicy = 'network-only';
+    if (this.options.fetchPolicy === 'no-cache') {
+      pollFetchPolicy = 'no-cache';
+    }
```

Comment:
```suggestion
    const { fetchPolicy } = this.options;
    // The fetchPolicy to be used when polling. Always network-only unless
    // this.options.fetchPolicy is no-cache (then polling uses no-cache too).
    const pollFetchPolicy: PollFetchPolicy =
      fetchPolicy === 'no-cache' ? fetchPolicy : 'network-only';
```
Mostly trying to use `const` instead of `let` here (personal style preference), but I think this comment also may be a little more accurate?